app/router.cjsx

```diff
@@ -237,7 +239,7 @@ module.exports =
       <Route path="mini-course" component={require './pages/lab/mini-course'} />
     </Route>
     <Route path="lab-policies" component={require './pages/lab/help/lab-policies'} />
-    <Route path="lab-how-to" component={require './pages/lab/help'} />
+    <Route path="lab-how-to" component={HowToBuildProject} />
```

Comment:
I don't think you'll ever need to hit this "/lab-how-to" route. It ends up pulling the `HowToBuildProject` without the help header, etc. The "/help" route, as defined by the `IndexRoute` component above, should be all you need here.